### ISSUE TITLE
AZ zip prefix wrong.

### ISSUE THREAD

**Issue by michaelkirk** [farski/TZip#1]
The AZ zip prefix is wrong. It's listed as 0,1,2 (same as eastern).

I think it should be 85,86 (based on  https://secure.wikimedia.org/wikipedia/en/wiki/Zipcode)

See the change on my rails 2 compatible fork:
https://github.com/Katlean/TZip/commit/65d53614104dd2ec6d9c2560094cb06df6408692

**Comment by farski** [comment]
So the problem with Arizona is that I don't think I could figure out the correct mapping of zipcode to time zones. There are a few very small areas that don't observe DST in AZ, so doing a catchall of 85**\* and 86**\* would result in inaccurate TZ info for some zips. When I was last working on this I was only implementing zonings if they were complete, which is why some are completely off. If you have a better understanding of which ZIPs in AZ are actually in the Arizona Timezone, and which are in Mountain time, I'd love some help there.

**Comment by michaelkirk** [comment]
My fix might not be right, but it's definitely closer than what was there before. 

Because there was overlap in the zip code prefixes (both az and eastern were assigned the same prefixes) the resolution of timezone was affected by their ordered storage in the hash, leading to some non-deterministic behavior. (e.g. sometimes a zip would resole to eastern, sometimes the same zip would resolve to az).

thanks for the code btw. =)

**Comment by farski** [comment]
Yeah, definitely doesn't make sense to have the wrong prefixes in there. I'm not going to put 85 and 86 in there in my repo because of the issues I mentioned; I'd rather those just fail than be wrong, until I figure out the actual values.
